api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -353,9 +354,9 @@ public void testLocalize_differentNamespace() throws Exception {
 
     Map<String, String> localizeMap = mapCaptor.getValue();
     assertThat(localizeMap).containsEntry(
-        "~/workspaces/proj:wsid/foo.ipynb", "gs://workspace-bucket/notebooks/foo.ipynb");
-    assertThat(resp.getClusterLocalDirectory()).isEqualTo("workspaces/proj:wsid");
-    JSONObject aouJson = dataUriToJson(localizeMap.get("~/workspaces/proj:wsid/.all_of_us_config.json"));
+        "~/workspaces/proj" + WORKSPACE_DELIMITER + "wsid/foo.ipynb", "gs://workspace-bucket/notebooks/foo.ipynb");
```

Comment:
opt_nit: I would recommend just hardcoding the exact string here for testing. It is a bit more maintenance to keep it in sync, but in this case I'd favor being explicit in showing the expected formatted value, rather than duplicating the path construction logic in the test.